package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "phantom-fe-codebandits",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "start": "webpack serve",
+    "build": "webpack"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/atlp-rwanda/phantom-fe-codebandits.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/atlp-rwanda/phantom-fe-codebandits/issues"
+  },
+  "homepage": "https://github.com/atlp-rwanda/phantom-fe-codebandits#readme",
```

Comment:
@samuel-shyaka-dus This line can create conflict when deploying to GitHub pages. The GitHub Pages package look for this entry **(homepage)** to associate the corresponding link to use and would be formatted as **[http://atlp-rwanda.github.io/phantom-fe-codebandits](http://atlp-rwanda.github.io/phantom-fe-codebandits)**

[Documentation linked here](https://www.c-sharpcorner.com/article/how-to-deploy-react-application-on-github-pages/#:~:text=add%20homepage%20property%20to%20package.json%20file)

Reply:
Just changed that